crates/typst-library/src/model/bibliography.rs

```diff
@@ -192,6 +193,48 @@ impl BibliographyElem {
     }
 }
 
+impl Packed<BibliographyElem> {
+    /// Produces the heading for the bibliography, if any.
+    pub fn realize_title(&self, styles: StyleChain) -> Option<Content> {
+        let span = self.span();
+        self.title
+            .get_ref(styles)
+            .clone()
+            .unwrap_or_else(|| {
+                Some(TextElem::packed(Packed::<BibliographyElem>::local_name_in(styles)))
+            })
+            .map(|title| {
+                HeadingElem::new(title)
+                    .with_depth(NonZeroUsize::ONE)
+                    .pack()
+                    .spanned(span)
+            })
+    }
+
+    /// Fetches and validates the bibliography works.
+    pub fn realize_works(
+        &self,
+        engine: &mut Engine,
+        styles: StyleChain,
+    ) -> SourceResult<std::sync::Arc<Works>> {
```

Comment:
Arc was already imported, no need to qualify.
```suggestion
    ) -> SourceResult<Arc<Works>> {
```